src/events/interactionCreate/checkInStart.js

```diff
@@ -1,14 +1,14 @@
 /**
- * Author: Arthur Powers
- * Date: 5/22/2024
+ * Author: Arthur Powers & Victoria Parsonage Ueda
+ * Date: 5/22/2024, 6/3/2024
```

Comment:
These can be deleted. Although nice for know who did what if there is an issue, it can be found via the commit history. Plus will be bloated as the project progresses and more people are included